### DISCUSSION TITLE
How to create multiple connections ?

### DISCUSSION THREAD

**Discussion by naveenbharadwaj19** [flutter-webrtc/flutter-webrtc#698] [Q&A]
I'm trying to add 10 participants to my webRTC . Each of them can share audio and video like Zoom or Gmeet . Problem is when i try to add 3rd participant getting black screen audio is not working. I'm using firestore as signaling server . Can someone point me to the right direction ?

**Comment by piyushChopra20** [comment]
> I'm trying to add 10 participants to my webRTC . Each of them can share audio and video like Zoom or Gmeet . Problem is when i try to add 3rd participant getting black screen audio is not working. I'm using firestore as signaling server . Can someone point me to the right direction ?

Hey have you gotten any solution for multiple peer connections for having a conference call over 3 peers?

**Reply by shivanshtalwar0** [reply]
Implementing SFU(selective forwarding unit) on client side using mesh of PeerConnections is never a good idea maybe you guys are looking for webrtc server like janus or jitsi

**Reply by piyushChopra20** [reply]
@shivanshtalwar0 What could be a better approach for peer to peer connection as for web in JS we can use Simple peer for handling multiple peer what could be the approach for flutter while using this package. The issue is my client doesn't want to pay for server and he is asking if it is possible via client end only we are only using Stun/turn server over AWS to have video calls but multiple peer is not possible based on the base library
So i m looking for an approach to make a peer to peer (group call) where client is handling most of the stuff(We using STOMP for broadcasting the handshakes)
